### ISSUE TITLE
CLI: port issues

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1816]
- [x] #1817
- [x] #1818
- [x] `--flag auto` to make the CLI automatically find a free port //later

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.7.1](https://github.com/open-wa/wa-automate-nodejs/compare/4.7.0...4.7.1) (2021-06-24)

- build(deps): bump parse-url from 5.0.3 to 5.0.5 [`#1814`](https://github.com/open-wa/wa-automate-nodejs/pull/1814)
- build(deps): bump swagger-ui-dist from 3.50.0 to 3.51.0 [`#1812`](https://github.com/open-wa/wa-automate-nodejs/pull/1812)
- Release 4.7.1 [`2037c61`](https://github.com/open-wa/wa-automate-nodejs/commit/2037c616fb7c252b728a79963fab0aff1d1fd767)
- 🔇cli: improve port taken log #1817 [`3d532ca`](https://github.com/open-wa/wa-automate-nodejs/commit/3d532ca4c48c41dce2746e78fbb6204924e31059)
- fix: #1818 [`5830615`](https://github.com/open-wa/wa-automate-nodejs/commit/58306158aa9c1ff2f35c101b376215e0fc53e9f4)
